bot/__main__.py

```diff
@@ -1,43 +1,36 @@
 import os
 
-from discord import Game
 from discord.ext.commands import when_mentioned_or
 
-from bot import config
-from bot.core.bot import Bot
+import config
+from core.bot import Bot
```

Comment:
This needs to be `bot.core.bot` since the code is being run from the root of the project by doing `py -m bot` or simply `pipenv run start` this file is itself is not being run directly so this import would fail, same goes fro `config` it needs to be `from bot import config`

Reply:
I'll trust you about that one